ios/brave-ios/Sources/BraveWallet/Crypto/Stores/SendTokenStore.swift

```diff
@@ -249,6 +264,11 @@ public class SendTokenStore: ObservableObject, WalletObserverStore {
         precisionAfterDecimalPoint: decimalPoint,
         rounded: rounded
       ).trimmingTrailingZeros
+    if amount == .all {
+      // `sendingMaxValue` reset to false in `sendAmount` didSet.
+      // Assign true after we've updated the value via 100% button.
+      sendingMaxValue = true
```

Comment:
Will it reset somehere after clicking `.all` then `.half` ?

Reply:
Yes it resets in the `didSet` for `sendAmount: String` (any input change to `sendAmount`): https://github.com/brave/brave-core/blob/9efbe9773f9b98bd81676d3c908d926003705c83/ios/brave-ios/Sources/BraveWallet/Crypto/Stores/SendTokenStore.swift#L52